packages/compass-aggregations/src/modules/explain.ts

```diff
@@ -256,16 +256,13 @@ const mapIndexesInformation = function (
   return explainIndexes
     .filter(x => x.index)
     .map((explainIndex) => {
-      const index = collectionIndexes.find(
+      const collectionIndex = collectionIndexes.find(
```

Comment:
Do we / should we test this function/module? I'm thinking it could help prevent any regressions.

Reply:
done :)